docker/nginx.dockerfile

```diff
@@ -0,0 +1,11 @@
+from nginx:latest
+
+RUN rm /etc/nginx/conf.d/*
+
+COPY nginx /etc/nginx
+
+RUN mkdir /etc/nginx/sites-enabled
+
+RUN cp /etc/nginx/sites-available/* /etc/nginx/sites-enabled/
+
+CMD ["nginx", "-g", "daemon off;"]
```

Comment:
I didn't test this but we might run into the same problem where if the container starts up before the other's are available nginx fails and then the whole system is down. I don't want to reconfigure nginx to start with failed hosts cause it's just not worth it for what this is, should we maybe just add like a 2-10 second sleep here to make sure the other servers are started?? Or maybe just run this in a while loop until it returns 0 (with like a `sleep 1` so it doesn't use 100% cpu)??

Reply:
can just add dependency on the docker file so it ensures that the router container is the last one to start